.github/CODEOWNERS

```diff
@@ -1312,6 +1313,10 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /x-pack/test_serverless/**/test_suites/common/management/transforms/ @elastic/ml-ui
 
 # Additional plugins and packages maintained by the ML team.
+/x-pack/test/functional/apps/aiops @elastic/ml-ui
+/x-pack/test/api_integration_basic/apis/aiops @elastic/ml-ui
+/x-pack/test/api_integration/services/aiops.ts @elastic/ml-ui
+/x-pack/test/api_integration/services/ml.ts @elastic/ml-ui
```

Comment:
This one should be moved into the `Machine Learning` section above.

Reply:
https://github.com/elastic/kibana/pull/197468/commits/54f7be2b2f6d5937378ef9428fb2de09e03fb5c3